src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -13,6 +13,28 @@ import {
     richTextEditorBuilder,
 } from './richTextEditor.es6';
 
+const initialVisibilityOptions = new Map();
+
+function setInitialVisibilityOfCheckboxes(id, element) {
+    const tableInForm = element.find('table').first();
+    const checkboxesInForm = tableInForm.find('tr').find('input.visibilityCheckbox');
+    const valuesOfCheckbox = new Map();
+    $.each(checkboxesInForm, (i, checkboxInForm) => {
+        const checkbox = `${$(checkboxInForm).attr('class')} - ${$(checkboxInForm).val()}`;
```

Comment:
Why the two spaces before and after the `-`? Also, if you want to use this kind of string as a key, you should probably abstract the key construction into another function, otherwise you risk typos.